client/js/navigation/DrawerScreen.js

```diff
@@ -0,0 +1,58 @@
+import React, {Component} from 'react';
+import {withNavigation} from 'react-navigation';
+import {SafeAreaView, Text, TouchableOpacity} from 'react-native';
+
+class DrawerScreen extends Component {
+  navigateToScreen = route => {
+    this.props.navigation.navigate(route);
+    this.props.navigation.closeDrawer();
+  };
+
+  render() {
+    return (
+      <SafeAreaView
+        style={{
+          backgroundColor: '#013859',
+          height: '100%',
+          width: '100%',
+          marginLeft: 0,
+        }}>
+        <TouchableOpacity
+          onPress={() => this.navigateToScreen('About')}
+          style={{
+            marginTop: 80,
+            borderBottomColor: 'white',
+            borderBottomWidth: 1,
+          }}>
+          <Text
+            style={{
+              fontSize: 20,
+              padding: 25,
+              color: 'white',
+            }}>
+            About Us
+          </Text>
+        </TouchableOpacity>
+        <TouchableOpacity
+          onPress={() => this.navigateToScreen('Contact')}
+          style={{borderBottomColor: 'white', borderBottomWidth: 1}}>
+          <Text
+            style={{
+              fontSize: 20,
```

Comment:
there's duplicate styling in this file (fontsize: 20, padding 25, color white), can you put this in Stylesheet.create?